extensions/azurecore/src/extension.ts

```diff
@@ -241,7 +241,10 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 			query: string): Promise<azurecore.ResourceQueryResult<T>> {
 			return azureResourceUtils.runResourceQuery(account, subscriptions, ignoreErrors, query);
 		},
-		onEncryptionKeysUpdated: eventEmitter!.event
+		onEncryptionKeysUpdated: eventEmitter!.event,
+		async getEncryptionKeys(): Promise<azurecore.CacheEncryptionKeys | undefined> {
+			return await providerService?.getEncryptionKeys();
```

Comment:
If the provider service isn't created that's going to break a log of other functionality, so similar to the above is it actually useful to try and handle it not being there?

Reply:
Handled.